lib/tools/adb-commands.js

```diff
@@ -4,6 +4,9 @@ import { getIMEListFromOutput, isShowingLockscreen, isCurrentFocusOnKeyguard,
 import path from 'path';
 import { fs } from '../utils.js';
 import net from 'net';
+import Logcat from '../logcat.js';
```

Comment:
for our local imports, let's do `from '../logcat';` not `from '../logcat.js';`, here and everywhere